english/forest/shelter/shelter.md

```diff
@@ -1,4 +1,8 @@
-You break down some branches and make a frame. You put your parachute over
-the top of it, tying it down securely with palm leaves. Luckily you did it 
-because as you make yourself comfortable for the night the heaven's open. 
-There's a terrible storm brewing... 
\ No newline at end of file
+You break down some branches and make a frame. You put your parachute over the top of it, the top of it, tying it down securely with palm leaves. Luckily you did ittying it down securely with palm leaves. Luckily you did it because as you make yourself comfortable for the night the heaven's open. There's a terrible storm brewing...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.